### DISCUSSION TITLE
Confused about comment about "fully native stack navigation"

### DISCUSSION THREAD

**Discussion by 1mike12** [infinitered/ignite#1822] [Q&A] [answered]
I see this comment that was part of a commit 10 months ago. 

**app.tsx**
```
// This puts screens in a native ViewController or Activity. If you want fully native
// stack navigation, use `createNativeStackNavigator` in place of `createStackNavigator`:
// https://github.com/kmagiera/react-native-screens#using-native-stack-navigator
```
but I don't see any usages of `createStackNavigator` and as far as I can tell, the project already uses the wix native stack navigator? Is this just a left-over comment or is the suggestion actually valid? What would the differences be of changing to the suggested module?

**Comment by leonskim** [comment, marked as answer]
Hey @1mike12!

`createStackNavigator` is used as default when generating a navigator by `ignite-cli`. For example,

### 1. Generate a navigator with `ignite-cli`
```
$ npx ignite-cli generate navigator TestNavigator 
```

### 2. Generated navigator
![carbon(1)](https://user-images.githubusercontent.com/8325407/143155256-77a0ae3e-7b20-4424-98d2-46a8a0af7abf.png)

So, after generating a navigator, you'll be able to turn that into `createNativeStackNavigator` if you need.
